tests/fixtures/dns-response-policy.tf

```
@@ -0,0 +1,22 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "dns-policy-existing" {
```

```suggestion
module "dns-response-policy" {
```

Can we keep full module path in the name? 

Also, if we add fixture creating dns-policy, I imagine it must exists? :-)

---

Thanks Wiktor for reviewing, renamed the module name. Yeah, we have dns-policy, Two examples adds rules to existing policy, so included it in fixtures.